cmd/repo-init/main.go

```diff
@@ -817,6 +817,8 @@ func determineWorkflow(workflow string, clusterProfile api.ClusterProfile) *stri
 			ret = "ipi-alibabacloud"
 		case api.ClusterProfileNutanix:
 			ret = "ipi-nutanix"
+		case api.ClusterProfileAWSOSDMSP:
+			ret = "awsosdmsp"
```

Comment:
No such workflow exists: unless you plan to write it, I think it's fine to not add it into the `repo-init` tool.